.prettierignore

```diff
@@ -1,4 +1,5 @@
 package.json
 .cache
 .public
-dist
\ No newline at end of file
+dist
+.percy-debug
```

Comment:
😄 

I ran format:js due to the prettier update and saw it was running on these files.

https://github.com/prettier/prettier/issues/2294